sklearn/ensemble/bagging.py

```diff
@@ -218,10 +219,14 @@ def __init__(self,
         self.bootstrap = bootstrap
         self.bootstrap_features = bootstrap_features
         self.oob_score = oob_score
+        self.warm_start = warm_start
         self.n_jobs = n_jobs
         self.random_state = random_state
         self.verbose = verbose
 
+        self.estimators_samples_ = []
+        self.estimators_features_ = []
```

Comment:
Those should be set in the fit. You might want to use `hasattr` to check if those variables are already allocated.